Realm/RLMNetworkTransport.mm

```diff
@@ -72,13 +72,10 @@ - (void)sendRequestToServer:(RLMRequest *) request
     }
     urlRequest.timeoutInterval = request.timeout;
 
-    RLMSyncManager *syncManager = RLMSyncManager.sharedManager;
-    RLMNetworkRequestOptions *options = syncManager.networkRequestOptions;
-
     for (NSString *key in request.headers) {
         [urlRequest addValue:request.headers[key] forHTTPHeaderField:key];
     }
-    id delegate = [RLMSessionDelegate delegateWithCertificatePaths:options.pinnedCertificatePaths
+    id delegate = [RLMSessionDelegate delegateWithCertificatePaths:@{}
```

Comment:
The delegate should just not have the certificate paths parameter if that functionality is no longer used.